tests/cogmisc/__init__.py

```diff
@@ -0,0 +1 @@
+
```

Comment:
is this file supposed to be here?

Reply:
Not sure, I added an __init__.py file because all test folders had one. And I changed the indentation to 4 spaces which probably explains the 1kb of memory.

Reply:
`cogmisc` should probably be `cogsmisc`